### PR TITLE
connect to db

### PR DESCRIPTION
**Pull request by arielleferre23**
add env secrets to ci

use sample.appsettings.json

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
+        run: dotnet restore
+
+      - name: Build
+        run: dotnet build --configuration Release --no-restore
+
+      - name: Test
+        run: dotnet test --no-restore --verbosity normal
+        env:
+          CONNECTION: ${{ secrets.CONNECTION }}
+          GITHUB: true
```
Comment (moondc) [inline .github/workflows/ci.yml:27]: I believe the github flag was only needed for the excel testing.  I'm not seeing it referenced in the code you're committing.

Comment (moondc) [reply .github/workflows/ci.yml:27]: nevermind

File: ETT-Backend/Configuration/AppConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+      serviceCollection.AddSingleton<IConfigurationRoot>(configuration);
+    }
+
+    public static string GetValue(string configurationKey)
+    {
+      string result = Environment.GetEnvironmentVariable(configurationKey.ToUpper());
+      if (String.IsNullOrEmpty(result))
+        result = configuration.GetSection(configurationKey).Value;
+      Console.WriteLine(result);
```
Comment (moondc) [inline ETT-Backend/Configuration/AppConfiguration.cs:37]: Remove console.logs

File: .github/workflows/ci.yml (hunk 1 above)
Comment (arielleferre23) [reply .github/workflows/ci.yml:27]: It's used in the configuration to point to the correct appsettings

File: ETT-Backend/Configuration/AppConfiguration.cs (hunk 2 above)
Comment (arielleferre23) [reply ETT-Backend/Configuration/AppConfiguration.cs]: done

File: ETT-Backend.Tests/Repository/DBConnectionTest.cs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      {
+        this.str = str;
+        this.num = num;
+      }
+    }
+
+    [Fact]
+    public void ShouldCreateDBConnectionObject()
+    {
+      DBConnection con = new DBConnection();
```
Comment (cdubya25) [inline ETT-Backend.Tests/Repository/DBConnectionTest.cs:24]: No assertion in this test, could add an Assert.NotNull but not a big deal either way

File: ETT-Backend/Repository/DBConnection.cs [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    {
+      get
+      {
+        return connection.State == System.Data.ConnectionState.Open;
+      }
+      private set { }
+    }
+    private NpgsqlConnection connection;
+
+    public DBConnection()
```
Comment (cdubya25) [inline ETT-Backend/Repository/DBConnection.cs:21]: So if we injected AppConfiguration at startup, we could just access it like this DBConnection(IAppConfiguration appConfiguration) (we would need an interface). That way the ServiceCollection in AppConfiguration is the same as throughout the app and not its own thing

Comment (arielleferre23) [reply ETT-Backend/Repository/DBConnection.cs:21]: Fixed AppConfiguration class

File: ETT-Backend.Tests/Repository/DBConnectionTest.cs (hunk 3 above)
Comment (arielleferre23) [reply ETT-Backend.Tests/Repository/DBConnectionTest.cs:24]: added :)

**Pull request merged**